### DISCUSSION TITLE
How can I change the print for frontmatter and mainmatter

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#425] [Q&A] [answered]
Hello professor.

I am formatting my thesis using the novathesis-7.0.0 template in Overleaf.

As I saw in the dispatch from UMinho, the norms indicate that, after the white title page, there must be on the next page (back of the title page), the statement regarding the conditions of use of the work by third parties (copyright), but at the moment what appears is the statement of integrity.
The UM dispatch should have this order:
- Reverse of the title page (copyright);
- sheet 2 - page 3 of the work (acknowledgments)
- next page must contain the Declaration of Integrity
- Next page must contain the title and abstract of the work
- ...

As I could verify, in the `uminho-defaults.ldf` file, `\ntsetprintorder{frontmatter}{`
it prints initially the copyright, then acknowledgments, and then the statement.
But in fact, what happens in the compilation is that it appears in another order:
![image](https://user-images.githubusercontent.com/98588105/230040733-06aaff9e-4954-49a2-88e2-3ef39cf5d91c.png)

After the title page, print the declaration of Integrity (statement), copyright, quote and acknowledgments.
How can I change this situation and order the way it is requested?

**Comment by joaomlourenco** [comment, marked as answer]
That's a bug… thanks for reporting it.
Easy fix:
Replace lines 27–37 from `NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf` with
```latex
% --------------------------------------------------------
% PRINT ORDER FOR frontmatter AND mainmatter
\printorder(frontmatter,final):={%
  copyright;       % (*) The copyright page
  acknowledgements;% (*) Print the acknowledgments
  statement;       % The statement page
  quote;           % (*) Print the quote
  abstracts;       % Print abstracts in multiple languages.
  alllistof;       % Print all the listof defined in “7_list_of.tex”
  printglossaries; % Print the Glossary, Acronyms, Symbols, etc…
}

\printorder(frontmatter,provisional):={\theprintorder(frontmatter,final)}
```

**Reply by Marcelo-Alves7** [reply]
Thanks professor.
Works as expected.
